jquery-idle-hands.js

```diff
@@ -205,43 +226,54 @@
         /**
          * Retrieves the session start time from local storage.
          *
-         * @return Number
+         * @return number
          */
         let getSessionStartTime = function () {
             return localStorage.get('sessionStartTime');
         }
 
-        /**
-         * Deletes the session start time from local storage.
-         */
-        let deleteSessionStartTime = function () {
-            localStorage.remove('sessionStartTime');
-        }
-
         /**
          * Sets the logout URL in local storage.
          *
-         * @return String
+         * @return string
          */
-         let setLogoutUrl = function (logoutUrl) {
+        let setLogoutUrl = function (logoutUrl) {
             localStorage.set('logoutUrl', logoutUrl);
-         }
+        }
 
         /**
          * Retrieves the logout URL from local storage.
          *
-         * @return String
+         * @return string
          */
-         let getLogoutUrl = function () {
+        let getLogoutUrl = function () {
             return localStorage.get('logoutUrl');
-         }
+        }
 
          /**
           * Clears values saved in local storage.
           */
-          let flushLocalStorage = function () {
+        let flushLocalStorage = function () {
             localStorage.flush();
-          }
+        }
+
+        /**
+         * Sets the logged out status in local storage.
+         *
+         * @param boolean
```

Comment:
following the convention in this file
```suggestion
         * @param boolean loggedOutStatus
```